src/api/PDFPage.ts

```diff
@@ -1143,6 +1145,8 @@ export default class PDFPage {
         color: options.color ?? undefined,
         borderColor: options.borderColor ?? undefined,
         borderWidth: options.borderWidth ?? 0,
+        dashArray: options.dashArray || [],
+        dashPhase: options.dashPhase ?? 0,
```

Comment:
Per my above comments, these two options will need to be made optional. I think that once we do that, I think it would be best to remove the `|| []` and `?? 0` defaults.

Reply:
Done, and added it like this: `borderDashArray: options.borderDashArray ?? undefined`, because this is how the rest of the optional parameters are handled. Although I'm not sure about the purpose of `?? undefined`, can you explain why that is needed?

Reply:
It shouldn't be necessary. I'm not actually sure how that wound in there. Maybe just the accidental result of some refactoring.

Reply:
No need to worry about removing them for now. I (or somebody else) will take them out later. ;)